src/Chapter1.hs

```diff
@@ -568,7 +571,11 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool 
+isVowel c  
+  | c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' = True
+  | c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U' = True
+  | otherwise = False
```

Comment:
Great implementation! To write it slightly shorter, you could use the [`toLower`](https://hackage.haskell.org/package/base-4.15.0.0/docs/Data-Char.html#v:toLower) function from the `Data.Char` module in `base`. But we cover imports only in the second chapter, so no action is required here for now 🙂

Reply:
Noted!